### PR TITLE
Cleanup of strings in Customer Center

### PR DESCRIPTION
**Pull request by vegaro**
Adds some strings that were not being localized

### CODE REVIEW

File: RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift [hunk 1]
```diff
@@ -236,13 +236,24 @@ private final class ManageSubscriptionPurchases: ManageSubscriptionsPurchaseType
...
+        case .day:
+            components.day = self.value
+        case .week:
+            components.weekOfMonth = self.value
+        case .month:
+            components.month = self.value
+        case .year:
+            components.year = self.value
+        default:
+            return NSLocalizedString("common_unknown", comment: "Unknown duration")
```
Comment (tonidero) [inline RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift:253]: Hmm I'm not sure I follow this, would we have a localized `common_unknown` key in the SDK? Ideally it should come from the backend, though we would need access to the localization data from here then...

**Review by tonidero — COMMENTED** [review]
Just a question, looks good otherwise!

File: RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift (hunk 1 above)
Comment (vegaro) [reply RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift:253]: this was just me mixing two things. This should just be `Unknown`. Good catch

**Comment by vegaro** [comment]
@tonidero found a bunch of other things that needed to be localised so I pushed those fixes here

**Pull request merged**